src/content/docs/style-guide/writing-guidelines/user-related-language-guidelines.mdx

```diff
@@ -72,13 +72,29 @@ We recommend linking to the [standard roles](/docs/accounts/accounts-billing/new
 [Default groups](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-group)      </td>
 
       <td>
-"When adding a user, add them to the [**Admin** group](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups)."
+Example use: 
+*When adding a user, you can add them to either the [**Admin** or **User** group](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups).*
  </td>
 
       <td>
-Sentence case, bold.
+Preferred: sentence case, bold. Alternatively, instead of bold, you can use italics or code-formatting: as long as it's clear that we're referring to something that has a definite, exact name. The words 'admin' and 'user' are generic, general words, so using formatting to indicate that we're referencing specific terms is important. 
+
+When referencing the default groups, we recommend linking to the [explanation of default groups](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups). 
+      </td>
 
-We recommend linking to the [explanation of default groups](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups) for more context. 
+    </tr>
+
+   <tr>
+      <td>
+[Capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#capabilities)
+      </td>
+
+      <td>
+Example use: 
+*The ability to do this is governed by the "modify APM settings" [capability](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#capabilities).*   </td>
+
+      <td>
+One way to reference a capability is by using quotes but we don't have firm rules on referencing the capabilities. One reason for this is that capabilities don't have specific proper names and are more informal. To see what we mean, see the screenshot of the capabilities UI table. The rows in the capabilities table have a heading (like "APM"), and refer to a feature (like "Application settings"), and have columns referencing more granular abilities (like "Create" or "Delete"). For this reason, we don't have firm rules for how to refer to a specific capability, other than to try to make it clear which capability you're referencing. 
       </td>
```

Comment:
Looks like you repeat `we don't have firm rules `in a couple places. Any way to make this less repetitious/condense the length for clarity?